app/mailers/account_transaction_report_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class AccountTransactionReportMailer < BaseMailer
```

Comment:
Let me that again. Initially I tried doing that, but there was an issue when delayed_job tried to de-serializing the report.